index.html

```diff
@@ -0,0 +1,571 @@
+  </code>
+</figure>
+
+The final two segments of the URL's [path](https://tools.ietf.org/html/rfc3986#section-3.3) **must** contain the feature's [default prefix](#prefix) and a [version tag](#versioning). The content of the URL up to and including the prefix—but excluding the version tag and trailing `/`—is the feature's *identity*. For the above example,
+<dl>
+  <dt markdown>`identity: "https://spec.example.com/a/b/c/exampleFeature"`</dt>
+  <dd>A global identifier for the feature. Processors can treat this as an opaque string identifying the feature (but not the version of the feature) for purposes of selecting an appropriate implementation.</dd>
+  <dt markdown>`name: "exampleFeature"`</dt>
+  <dd markdown>The feature's name, for purposes of [prefixing](#prefixing) schema elements it defines or extends.</dd>
+  <dt markdown>`version: "v1.0"`</dt>
+  <dd markdown>The tag for the [version](#versioning) of the feature used to author the document. Processors **must** select an implementation of the feature which can [satisfy](#satisfaction) the specified version.</dd>
+</dl>
+
+The version tag **must** satisfy the [BNF grammar for `‹version tag›`](#backus-naur-form-for-versions-and-version-tags).
+
+### Ignore meaningless URL components
+
+When extracting the URL's `name` and `version`, processors **must** ignore any url components which are not assigned a meaning. This spec assigns meaning to the final two segments of the [path](https://tools.ietf.org/html/rfc3986#section-3.3). Other URL components—particularly query strings and fragments, if present—**must** be ignored for the purposes of extracting the `name` and `version`.
+
+<figure>
+  <figcaption>
+    <a name=feature-url-ignore-meaningless href=#feature-url-ignore-meaningless class=a-header>Ignoring meaningless parts of a URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>https://example.com/<span>exampleSpec<aside>name</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span><span class=grey>?key=val&k2=v2#fragment<aside>ignored</aside></span>
+  </code>
+</figure>
+
+### Why is versioning in the URL, not a directive argument?
+
+The version is in the URL because when a human reader visits the URL, we would like them to be taken to the documentation for the *version of the feature used by this document*. Many text editors will turn URLs into hyperlinks, and it's highly desirable that clicking the link takes the user to the correct version of the docs. Putting the version information in a separate argument to the `@core` directive would prevent this.
+
+## Prefixing
+
+With the exception of a single root directive, core feature specifications **must** prefix all schema elements they introduce. The prefix:
+1. **must** match the default name of the feature as derived from the feature's specification URL,
```

Comment:
Now you're saying "default name". Pick one of "name", "default prefix", and "default name" and stick with it.  (There are contexts where it might make sense to write "the default prefix when there is no `as:` is the feature's name", but that's different from talking about "the default prefix of the feature" as if it has an attribute called "default prefix".)